### PR TITLE
[docs] Add links to examples directory in Github repo.

### PR DESCRIPTION
**Pull request by travelton**
This directory in Github provides several useful configuration files that are worth mentioning in the Quick Start and Admin guides.

### CODE REVIEW

File: docs/4.2/admin-guide.md [hunk 1]
```diff
@@ -188,8 +188,10 @@ Docs](cli-docs.md#teleport-start) or run `teleport start --help`
 
 ### Configuration File
 
-Teleport uses the YAML file format for configuration. A sample configuration
-file is shown below. By default, it is stored in `/etc/teleport.yaml`
+Teleport uses the YAML file format for configuration. A sample configuration file is shown below. By default, it is stored in `/etc/teleport.yaml`.
```
Comment (benarent) [inline docs/4.2/admin-guide.md:191]: One of the core issues with this sentence, is that we say  `a sample configuration file`. What we have below is really a reference.  We need to split this up, ( not necessarily in this PR ) but I'll DM you another project I'm working on @travelton

File: docs/4.2/quickstart.md [hunk 2]
```diff
@@ -22,10 +22,10 @@ VM.
   the best place to start!
 
 This guide is only meant to demonstrate how to run Teleport in a sandbox or demo
-environment, and showcase a few basic tasks you can do with Teleport.
+environment, and showcase a few basic tasks you can do with Teleport. For additional configuration examples, see [teleport/examples](https://github.com/gravitational/teleport/tree/master/examples), within the Teleport Github repository.
 
-**You should not follow this guide if you want to set up Teleport in production.
-Instead follow the [Admin Guide](admin-guide.md)**
+!!! note "IMPORTANT"
+    You should not follow this guide if you want to set up Teleport in production. Instead follow the [Admin Guide](admin-guide.md).
```
Comment (benarent) [inline docs/4.2/quickstart.md:28]: Let's point to https://gravitational.com/teleport/docs/production/ instead.

File: docs/4.2/admin-guide.md (hunk 1 above)
Comment (travelton) [reply docs/4.2/admin-guide.md:191]: Agreed. It's information overload. I'm hoping the inclusion of the sample configurations lead folks to the simplified samples.

File: docs/4.2/quickstart.md (hunk 2 above)
Comment (travelton) [reply docs/4.2/quickstart.md:28]: Good idea, will do!

**Comment by webvictim** [comment]
A bunch of the stuff in the `examples` directory is out of date IIRC, we should definitely take a sweep over it and improve.

**Review by benarent — APPROVED** [review]

**Comment by benarent** [comment]
@webvictim I agree, we should also go back and review our `examples` but I've go ahead and merged this for now.

**Comment by travelton** [comment]
Similarly, I will add this to my to-dos! Thanks all!

**Pull request merged**